docs/rules/trials.md

```diff
@@ -0,0 +1,80 @@
+* Compound Question: A question that contains two or more separate questions and requires multiple answers.
+* Hearsay: Testimony made about a statement made by someone other than the witness.
+* Inflammatory: The question is intended to cause the jury/judge to view a subject in an unreasonable prejudiced way. For example: asking a witness about supposed past crimes that never happened.
+* Non-Responsive Answer: The answer given does not respond to the question asked.
+* Leading Question: The question asked to the witness suggests its own answer. For example: "Is it true that you saw the defendant in the Engineering lobby". Such questions however may be asked in cross-examination or if the witness is openly antagonistic towards the party that called them to testify.
+* Relevance: The question is irrelevant to the case.
+* Vague Question: A question that is incomprehensible, incomplete, or leads to an ambiguous answer.
+* Foundation: The question relates to matters that the witness' personal experience does not extend to.
+* Speculation: The witness is asked to speculate or guess the answer.
+* Privilege: The witness may not legally answer a question. For example the captain cannot state information that has been classified by Nanotrasen.
+
+
+**Evidence objections**: These are objections specific to certain piece of evidence:
+*Best Evidence: The original source of evidence must be used as evidence when available. Otherwise evidence may be used if it can be authenticated for example a stamped copy of a document.
+*Fruit of the Poisonous tree: The evidence was obtained illegally.
+*Incomplete: The provided evidence is incomplete either because some of it was lost, or the party presenting it is withholding a part of it. For example a section of a document taken out of context.
+*Lack of Foundation: The evidence cannot be authenticated.
+
+Objections are to be adjudicated by the presiding Judge.
+
+###Mistrials
+
+A mistrial occurs when there has been a serious error in the trial proceedings, or a jury is unable to reach a verdict. In such cases the trial is rendered null and void and a retrial may occur. This may be due to several causes such as:
+*Conflict of interest: The Judge, or the Jury are incapable of impartially performing their duties, or if the attorneys have a conflict of interest against their own client.
+*Hung Jury: The Jury was unable to decide on a verdict in a reasonable amount of time.
+*Disqualification: A member of the jury was disqualified after being selected and the parties don't wish to proceed with the remaining jurors.
+*Improperly admitted evidence: Unlawful evidence was submitted during the trial.
+*Misconduct: The actions of a party, juror, or an outside actor have made the carrying out of due process impossible.
+*Absence of a vital individual: The absence of a vital member of the trial such as a lawyer, or the defendant due to death or other means that make it impossible for them to appear.
```

Comment:
Missing space before the text begins (i.e. `- Test` instead of `-Test`), Change list items to use `-` instead of `*`,